articles.tex

```diff
@@ -680,4 +680,39 @@
 The Judicial Board may present an official ruling following the Judicial Investigation.
 This ruling may be appealed in the same manner as an Executive Board decision \ref{Appeals}.
 
+% ARTICLE VII - TEMPORARY AMENDMENTS
+\article{Temporary Amendments}
+In the case that an amendment should be temporary, the amendment's changes should be listed here. 
+Each of these amendments is given a designated period of time, and will cease to exist immediately following the end of the period.
+If at any point interpretation of the changes needs to be made, the Executive Board will use the process defined in \ref{Judicial} to come to an agreement on how to proceed. 
+Occasionally, Temporary Amendments may become permanent through a referendum, which is described in detail as part of each Temporary Amendment. 
+\asection{Changes to Evaluations}
+During the 2018-2019 period the following will supercede the text otherwise found in this document.
+\asubsection{Changes}
+\asubsubsection{Removal of Introductory Project}
+\ref{The Introductory Project} will no longer be a requirement as described in \ref{The Introductory Process}. 
+Additionally, \ref{Introductory Executive Board} will also no longer be elected or perform any duties during the described period. 
+\asubsubsection{Change of Packet Percentage}
+\ref{Creation of Accounts} currently stipulates that Introductory Members must acquire 60\% (rounded up to the nearest whole person) of required signatures (excluding those of Resident members who have not passed a Membership Evaluation) in the Introductory Packet or successfully complete Introductory Evaluations. 
+During the described period, the percentage of required signatures in the Introductory Packet is increased to 75\%. 
+\asubsubsection{Changes to Packet Times}
+Introductory Members will recieve \ref{The Introductory Packet} following the House Meeting or Evaluations Directorship meeting of the second week of each semester. 
```

Comment:
So some intro members will have a packet a few days longer than others, depending on if they start at house meeting or evals?

Reply:
Some, not many, would have 1-3 less days than the rest of their peers. During this period people will know this and the applicants will still be able to do packet talks. 1-3 days isn't a concern for me, and I think we're still being pretty fair.

Reply:
Yep, I don't think it's a huge deal either. Just wanted to make sure that was intentional.